.github/workflows/build-pdb.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Build PDB
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml -DskipTests
```

Comment:
good stuff... Shouldn't we let tests run?

Reply:
Glad you asked :smile: 
I separated that into more actions to have it segregated: https://github.com/feedzai/pdb/pull/157

Reply:
See in example how Pulse separates things: https://github.com/apache/pulsar/actions

Reply:
And with `package` it will also run the tests right?

Reply:
But won't the new workflow to run the tests also have to compile the code?

Reply:
Yes they do have to compile the code. It's not a big deal, compiling PDB is 4 seconds. We can use caching also. But I'll have to check how it works inter-workflow.

Reply:
@TiagoMSSantos This is now using a cache. Let's see how it'll behave in the future workflows.

Reply:
Ok then, it looks good to me.